config.yaml

```
@@ -490,6 +490,9 @@ mgmtworker:
     # Sets the interval between retry attempts in seconds.
     task_retry_interval: 15
 
+  # A local copy of Cloudify resources (e.g. deployment working directories)
+  resources_root: /tmp/resources
```

```suggestion
  resources_root: /opt/manager/resources
```
This needs to stay `/opt/manager/resources` because plugins write there